### DISCUSSION TITLE
Certificate expired again

### DISCUSSION THREAD

**Discussion by backgroundprocessing** [SpenceKonde/ATTinyCore#865] [Support / Q & A]
Hi Spence,
sorry to hassle you but the certificate for https://drazzy.com has expired again, so the ATTinyCore cannot be installed.
Would it be possible to have the board manager package file on Github instead of at drazzy.com?

Thanks for the great work, Bruce.

**Comment by hmeijdam** [comment]
#752

**Reply by lindmeira** [reply]
What's that #752 supposed to be? The guy's just rambling and the certs for drazzy.com are indeed expired again (since May 28)! Is there an alternative link already or not?

**Reply by ConzZah** [reply]
made a workaround:
https://github.com/ConzZah/ATTinyCore/blob/v2.0.0-devThis-is-the-head-submit-PRs-against-this/package_drazzy.com_index.json

hope this helps :)

**Reply by lindmeira** [reply]
It does help. Thanks.

**Comment by sleemanj** [comment]
Workarounds exist:




https://github.com/SpenceKonde/megaTinyCore/issues/1059#issuecomment-1950286469



https://github.com/SpenceKonde/megaTinyCore/issues/1059#issuecomment-1950286469



there is a current Pull Request also to include the json right here in the repo so that people could just add that url and not deal with the perpetually broken drazzy.com, I think Spence has a moral objection or something to using Github to host it, but I must admit that Spence's verbose ticket replies sometimes don't get read very deeply by me so perhaps I am mistaken.
